servers/src/test/java/tachyon/worker/block/TieredBlockStoreTests.java

```diff
@@ -335,4 +335,32 @@ public void abortCommitedBlockTest() throws Exception {
     mBlockStore.commitBlock(USER_ID1, TEMP_BLOCK_ID);
     mBlockStore.abortBlock(USER_ID1, TEMP_BLOCK_ID);
   }
+
+  @Test
+  public void commitBlockTwiceTest() throws Exception {
+    mThrown.expect(AlreadyExistsException.class);
+    mThrown.expectMessage(
+        "checkTempBlockOwnedByUser failed: blockId " + TEMP_BLOCK_ID + " is committed");
+    TieredBlockStoreTestUtils.createTempBlock(USER_ID1, TEMP_BLOCK_ID, BLOCK_SIZE, mTestDir1);
```

Comment:
In the other tests you usually put an empty line between the test setup and actual test operations?